planck/PLKCommandLine.m

```diff
@@ -222,6 +223,7 @@ +(int)processArgsCount:(int)argc vector:(char * const *)argv
                 staticFns = YES;
                 break;
             }
+            case 'q':
             case 'z':
             {
                 noBanner = YES;
```

Comment:
This could be revised to be `quiet` instead of `noBanner` (thinking that it will have broader use)